src/assets/native.ts

```diff
@@ -55,15 +55,28 @@ const nativeAssets: { [index: string]: Asset } = {
     color: '#000000',
     decimals: 24
   },
-  POLYGON: {
-    name: 'Polygon',
+  MATIC: {
+    name: 'Matic',
     chain: ChainId.Polygon,
     type: 'native',
-    code: 'POLYGON',
+    code: 'MATIC',
     coinGeckoId: 'matic-network',
     color: '#8247E5',
     decimals: 18
+  },
+  ARBETH: {
+    name: 'Arbitrum ETH',
+    chain: ChainId.Arbitrum,
+    type: 'native',
+    code: 'ARBETH',
+    coinGeckoId: 'ethereum',
+    color: '#28A0EF',
+    decimals: 18
   }
 }
 
-export default nativeAssets
+const nativeCodeSynonyms = {
```

Comment:
What is the purpose of this?  It can be included in the Asset type as an optional parameter

Reply:
It is needed, because eth on arbitrum and eth on main ethereum network have the same coingeckoId and this is causing problems in the wallet and agent. However, you are right, I can try to refactor this and to update the asset type

Reply:
it sounds like this is actually a problem in the coingecko clients in wallet + agent?

Reply:
Yes, we can say so - coingecko do not have a ticker of eth on Arbitrum, because it is basically eth, it is just migrated on this second layer solution

Reply:
Ok, what i mean is probably this property is not required at all, the wallet/agent should be able to handle 2 assets with the same coingeckoID fine. you can give it a try